go/vt/vtgate/engine/fake_primitive_test.go

```diff
@@ -81,6 +92,30 @@ func (tp *fakePrimitive) StreamExecute(vcursor VCursor, bindVars map[string]*que
 	return nil
 }
 
-func (tp *fakePrimitive) GetFields(vcursor VCursor, bindVars map[string]*querypb.BindVariable) (*sqltypes.Result, error) {
-	return tp.Execute(vcursor, bindVars, false /* wantfields */)
+func (f *fakePrimitive) GetFields(vcursor VCursor, bindVars map[string]*querypb.BindVariable) (*sqltypes.Result, error) {
+	f.log = append(f.log, fmt.Sprintf("GetFields %v", printBindVars(bindVars)))
+	return f.Execute(vcursor, bindVars, true /* wantfields */)
+}
+
+func (f *fakePrimitive) ExpectLog(t *testing.T, want []string) {
+	t.Helper()
+	if !reflect.DeepEqual(f.log, want) {
+		t.Errorf("vc.log:\n%v\nwant:\n%v", strings.Join(f.log, "\n"), strings.Join(want, "\n"))
```

Comment:
nit - I found it easier to read when we have: `vc.log got:\n%v\nwant:\n%v`

Reply:
Done.